app/src/location/location.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { LocationController } from './location.controller';
+
+describe('LocationController', () => {
+  let controller: LocationController;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [LocationController],
+    }).compile();
+
+    controller = module.get<LocationController>(LocationController);
+  });
+
+  it('should be defined', () => {
+    expect(controller).toBeDefined();
```

Comment:
Would you consider the test coverage of this application production-ready? If so (or if not), what is the usual coverage % you go for on such applications?

Reply:
this is a simple application so we can have 90% coverage here. but I am not implemented tests